### PR TITLE
Plumbing - Cleanup old onboarding files

### LINKED ISSUE #3311: Remove old onboarding files
Now that we've shipped the onboarding update, we can clean up old onboarding flow related files + instances where we checked for `SUPPORT_TABBED_ONBOARDING`

### PR DESCRIPTION
**Pull request by hyphenized**
Closes #3311
Closes https://github.com/tahowallet/extension/issues/2630

## To Test
- [x] Check onboarding still works as usual
- [x] Removing all accounts should open up the onboarding tab
- [x] Onboarding tab should open post install

Latest build: [extension-builds-3314](https://github.com/tahowallet/extension/suites/13144497149/artifacts/714580398) (as of Thu, 25 May 2023 12:15:49 GMT).

### CODE REVIEW

File: ui/index.ts [hunk 1]
```diff
@@ -38,25 +37,23 @@ export async function attachTabUIToRootElement(): Promise<void> {
...
+    // rather than rendering the popup
+    const baseURL = browser.runtime.getURL("tab.html")
+    const tabs = (await browser.tabs.query({ url: baseURL })).filter(
+      (tab) => tab?.url && tab.url.includes("onboarding")
+    )
+    if (tabs.length > 0 && tabs[0]?.id) {
+      await browser.tabs.update(tabs[0].id, { active: true })
+    } else {
+      await browser.tabs.create({
+        url: browser.runtime.getURL("tab.html#onboarding"),
```
Comment (jagodarybacka) [inline ui/index.ts:53]:
```suggestion
        url: browser.runtime.getURL(ONBOARDING_ROOT),
```

File: ui/pages/Tab.tsx [hunk 2]
```diff
@@ -1,25 +1,17 @@
...
 
-const isTabbedOnboarding = isEnabled("SUPPORT_TABBED_ONBOARDING")
-
 /**
  * Entry point for UI shown in browser tabs.
  */
 export default function Tab({ store }: { store: Store }): ReactElement {
-  const Container = isTabbedOnboarding
-    ? BrowserTabContainer
-    : BrowserTabContainer.Legacy
```
Comment (jagodarybacka) [inline ui/pages/Tab.tsx:21]: We can remove Legacy version from `BrowserTabContainer` component

File: ui/index.ts (hunk 1 above)
Comment (jagodarybacka) [reply ui/index.ts:53]: also here: https://github.com/tahowallet/extension/blob/onboarding-cleanup/src/background.ts#L10

Comment (jagodarybacka) [reply ui/index.ts:53]: we can remove flag: https://github.com/tahowallet/extension/blob/onboarding-cleanup/.env.defaults#L21

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
Minor fixes requested ⬇️ 

Also I would vote to make it a separate release as this is -2k lines and I'm afraid of how it will affect review time

**Review by kkosiorowska — COMMENTED** [review]
I think it's also a good idea to find and next close the issues for old onboarding. One of them:

https://github.com/tahowallet/extension/issues/2630

File: ui/index.ts (hunk 1 above)
Comment (kkosiorowska) [reply ui/index.ts]: Fixed.

**Review by kkosiorowska — APPROVED** [review]
LGTM :white_check_mark:

**Pull request merged**